test/unit/lodash.spec.js

```diff
@@ -0,0 +1,721 @@
+                throw new Error(message);
+            };
+            attempt = _.attempt(fun);
+        });
+        it('should verify params', function () {
+            expect(typeof message).to.eql('string');
+            expect(message).to.eql('It is simple error!');
+        });
+        it('should throw error object ', function () {
+            expect(_.isError(attempt)).to.eql(true);
+        });
+    });
+
+    describe('identity', function () {
+        var params;
+        before(function(){
+            params = datasets.identity();
+        });
+        it('should check type', function(){
+            var elem = params[0];
+            expect(elem instanceof Object).to.eql(true);
+        });
+        it('should identity object', function(){
+            var identity = _.identity.apply(_, params);
+            expect(identity).to.have.property('eggs', 2);
+            expect(identity).to.have.property('flour', true);
+        });
+    });
+
+    describe('method', function () {
+        var params,
+            objects,
+            elem1,
+            method,
+            result;
+        before(function () {
+            params = datasets.method();
+            elem1 = params[0];
+            objects = [
+                { 'a': { 'b': _.constant(2) } },
+                { 'a': { 'b': _.constant(1) } }
+            ];
+            method = _.method.apply(_, params);
+            result = _.map(objects, method);
+        });
+        it('should verify params', function () {
+            expect(params).to.have.length(1);
+            expect(typeof elem1).to.eql('string');
+            expect(elem1).to.have.length(3);
+        });
+        it('should returns the new invoker function', function () {
+            expect(result).to.eql([2,1]);
+        });
+    });
+
+    describe('methodOf', function () {
+        var params,
+            array,
+            obj,
+            methodOf,
+            result;
+        before(function () {
+            array = _.times(3, _.constant);
+            params = datasets.methodOf(array);
+            obj = params[0];
+            methodOf = _.methodOf.apply(_, params);
+            result = _.map(['a[2]', 'c[0]'], methodOf);
+        });
+        it('should verify params', function () {
+            expect(params).to.have.length(1);
+            expect(obj instanceof Object).to.eql(true);
+            expect(Object.keys(obj)).to.have.length(3);
+        });
+        it('should returns the new invoker function', function () {
+            expect(result).to.eql([2,0]);
+        });
+    });
+
+    describe('noop', function () {
+        var params;
+        before(function(){
+            params = datasets.noop();
+        });
+        it('should check types', function(){
+            expect(params[0] instanceof Object).to.eql(true);
+        });
+        it('should check properties obj', function(){
+            var elem1 = params[0];
+            expect(elem1).to.have.property('number', 5);
```

Comment:
You shouldn't check exactly input. If you don't know how to test some lodash function we should think to leave it (or find a way to test)